### PR TITLE
Add instructions for one-line installation

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

We use `devel` as the image tag for the `main` branch.

### CODE REVIEW

File: docs/setup/install.md [hunk 1]
```diff
@@ -8,19 +8,37 @@ Make sure the following conditions are met:
...
 ```sh
 git clone https://github.com/kubernetes-sigs/kueue.git
 cd kueue
 IMAGE_REGISTRY=registry.example.com/my-user make image-build image-push deploy
 ```
 
-The controller will run in the `kueue-system` namespace.
-
-## Uninstall Kueue
+### Uninstall Kueue
```
Comment (ahg-g) [inline docs/setup/install.md:41]:
```suggestion
### Uninstall
```

File: README.md [hunk 2]
```diff
@@ -22,15 +22,18 @@ presents the detailed design of the controller.
...
-```sh
-IMAGE_REGISTRY=registry.example.com/my-user make image-build image-push deploy
+```shell
+kubectl apply -k github.com/kubernetes-sigs/kueue/config/default
 ```
 
-The controller will run in the `kueue-system` namespace.
+The controller runs in the `kueue-system` namespace.
+
+Read the [full installation guide](/docs/setup/install.md) to learn more.
```
Comment (cortespao) [inline README.md:36]:
```suggestion
Read the [installation guide](/docs/setup/install.md) to learn more.
```

**Review by cortespao — APPROVED** [review]
Great work 😄

File: README.md (hunk 2 above)
Comment (alculquicondor) [reply README.md]: Done

File: docs/setup/install.md [hunk 3]
```diff
@@ -8,19 +8,37 @@ Make sure the following conditions are met:
...
+To install the latest development version of Kueue in your cluster, run the
+following command:
+
+```shell
+kubectl apply -k github.com/kubernetes-sigs/kueue/config/default
+```
+
+The controller runs in the `kueue-system` namespace.
+
+### Uninstall Kueue
```
[the remark this answers is not in the document]
Comment (alculquicondor) [reply docs/setup/install.md]: Done

File: docs/setup/install.md (hunk 1 above)
Comment (alculquicondor) [reply docs/setup/install.md]: Done

**Pull request merged**